docs/changelog.md

```diff
@@ -36,6 +36,24 @@ It doesn't include
 
 ## Unreleased
 
+### Highlights for users
+
+* (Android) You can now share content from other apps to Zulip. (#117)
+* Fixed a bug where network or server issues could cause an infinite
+  full-screen loading spinner. It now times out after 60 seconds. (#4165)
+* Search screen now opens at the newest messages, not the oldest. (#4858)
+* "Do not mark messages read on scroll" option is now more discoverable.
+  (#4850)
+* Basic support for polls. More to come! (#3205)
+
+### Highlights for developers
+
+* All object types are now exact or explicitly inexact. (#3452)
+* We're now using @react-native-community/push-notifications-ios instead of
+  two different libraries. (#4115)
+* Resolved issues (latest to earliest): #117, #4165, #4858, #4850, #4849,
+  #3205, part of #4309 (PR #4817), #3244, #3452, parts of #4540 and #2366
```

Comment:
Also PR #4635, which added a (small) user-facing feature that we don't seem to have filed an issue for. (Goes right next to #3244, which was fixed in the same PR.)

And between #4115 and #4657, there's
  PR #4821 shorter debounce on mark-as-read
  PR #4820 fix a BAD_REQUEST 
  PR #4815 rm "Possible infinite loop"
  PR #4797 turn off lazy loading in MainTabsScreen
Those didn't have GitHub issues filed for them, but they each fixed a bug or made a small improvement in user-visible behavior.